package.json

```diff
@@ -12,27 +12,32 @@
     "generate:pagopa-api-walletv2": "rimraf generated/definitions/pagopa/walletv2 && mkdir -p generated/definitions/pagopa/walletv2  && gen-api-models --api-spec $npm_package_pagopa_api_walletv2 --out-dir ./generated/definitions/pagopa/walletv2 --no-strict --request-types --response-decoders",
     "lint": "eslint . -c .eslintrc.js --ext .ts,.tsx",
     "lint-autofix": "eslint . -c .eslintrc.js --ext .ts,.tsx --fix",
-    "test": "jest --detectOpenHandles",
+    "test": "jest --detectOpenHandles --runInBand",
```

Comment:
@infantesimone thank you for pointing out. It should be safe to run the tests in parallel as each `it` block uses its own instance of browser and stub server. Nevertheless, in my opinion, removing `runInBand` will not give the expected performance improvement, since `jest`, after the collection phase, as I understand from the linked doc, runs the tests serially. 
https://jestjs.io/docs/en/setup-teardown.html#order-of-execution-of-describe-and-test-blocks
Moreover, in pipeline environment the number of test workers is usually 1.

Reply:
@infantesimone  This is the comparison on my workstation

```
$ jest --detectOpenHandles

Test Suites: 6 passed, 6 total
Tests:       25 passed, 25 total
Snapshots:   0 total
Time:        6.026 s
Ran all test suites.
Done in 11.71s.

$ jest --detectOpenHandles --runInBand

Test Suites: 6 passed, 6 total
Tests:       25 passed, 25 total
Snapshots:   0 total
Time:        5.27 s, estimated 6 s
Ran all test suites.
Done in 10.06s.
```